src/MLDataPattern.jl

```diff
@@ -54,6 +54,12 @@ export
     eachobs,
     eachbatch
 
+simplify_obsdim(::ObsDim.First) = :first
```

Comment:
I don't think simplify is the right word.
`simplify_obsdim` to me says the output should also be an `ObsDim` but a simpler one.

Possibly   `obsdim_string` ?